src/torchcodec/_core/CudaDeviceInterface.cpp

```diff
@@ -401,127 +325,39 @@ void CudaDeviceInterface::convertAVFrameToFrameOutput(
   // also need to check that the AVFrame is in AV_PIX_FMT_NV12 format (8 bits),
   // because this is what the NPP color conversion routines expect. This SHOULD
   // be enforced by our call to maybeConvertAVFrameToNV12OrRGB24() above.
```

Comment:
Below:

Note how we removed the `if (avFrame->hw_frames_ctx != nullptr) ...` which were needed because this could be called from the Beta interface, where `avFrame->hw_frames_ctx` would be null. Now we don't need to check for that anymore.